eng/ci/official-build.yml

```diff
@@ -47,7 +47,7 @@ extends:
     sdl:
       codeql:
         # Move codeql for source languages to source analysis stage
-        runSourceLanguagesInSourceAnalysis: true
+        # runSourceLanguagesInSourceAnalysis: true # temporary change to clean up stale python resuls
```

Comment:
Can we have a github issue (may be in "secure default" tab) to track revert for this temporary change? And adding a link to that issue here would be helpful.

typo : `results`

Reply:
Opened #10515 10515